interview-questions.md

```diff
@@ -8,42 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Compare and contrast JavaScript and Ruby. What are three things they have in common and what are three differences?
 
+I had to look this up so the similarities and differences is my researched answer
+
 Similarities
 
-1.
-2.
-3.
+1. Interpreted languages: code is executed by an interpreter at runtime which allows developers to write and see the results quickly. 
+2. Dynamic typing: variable types are determined at runtime rather than at compile time and variables can hold all types of data.
+3. Object oriented: both support OOP, this allows develpers to create objects that has data and functions related to that data.
```

Comment:
Ruby is definitely OOP but technically JavaScript uses prototypical inheritance.